go.mod

```diff
@@ -1,32 +1,40 @@
 module github.com/google/generative-ai-go
 
-go 1.21
+go 1.22
```

Comment:
We'd like to support the most recent two versions of Go. That is the Go team's standard policy.